docs/README.md

```diff
@@ -13,21 +13,9 @@
   <a href="https://github.com/uber/deck.gl/blob/6.0-release/docs/README.md">
     <img src="https://img.shields.io/badge/deck.gl-v6.0-green.svg?style=flat-square" />
   </a>
-  <a href="https://github.com/uber/deck.gl/blob/5.3-release/docs/README.md">
```

Comment:
While you are cleaning this up, should we also:
- Add 7.0 to "These docs are for" section.
- Add 6.4 to "Looking for an old version?" section.

Reply:
Good catch, done : 

<img width="839" alt="Screen Shot 2019-04-09 at 11 52 12 AM" src="https://user-images.githubusercontent.com/9502731/55827076-3cb08c00-5abe-11e9-9d14-2a7d31c3d81a.png">

Reply:
NOTE: link v7.0 will be broken until we branch `7.0-release`.